src/style.css

```
@@ -0,0 +1,48 @@
+body {
+  margin: 0;
+  --clear-star: url(images/star.svg);
+  --golden-star: url(images/star-active.svg);
+}
+
+.stars {
+  display: flex;
+  height: 15px;
+}
+
+.star {
+  width: 15px;
+  margin-right: 4px;
+  background: var(--clear-star);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--1 :not(:nth-child(n+2)) {
+  background: var(--golden-star);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--2 :not(:nth-child(n+3)) {
+  background: var(--golden-star);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--3 :not(:nth-child(n+4)) {
+  background: var(--golden-star);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--4 :not(:nth-child(n+5)) {
+  background: var(--golden-star);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--5 > * {
+  background: var(--golden-star);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

then combine all these rules into one like this
```suggestion

.stars.stars--3 :nth-child(something),
.stars.stars--4 :nth-child(something),
.stars.stars--5 :nth-child(something) {
  background: var(--golden-star);
}
```